packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js

```diff
@@ -0,0 +1,30 @@
+const guidancePagesController = require('../../../src/controllers/guidance-pages');
+const { mockReq, mockRes } = require('../mocks');
+const { VIEW } = require('../../../src/lib/views');
+
+describe('controllers/appellant-submission/appeal-statement', () => {
+  let req;
+  let res;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+    jest.resetAllMocks();
+  });
+
+  describe('getBeforeAppeal', () => {
```

Comment:
This tests happy path, does it need testing to see if the correct error page displays if anything goes wrong?

Reply:
We don't test this for anything else so would suggest not